src/app/components/Headline/index.jsx

```diff
@@ -17,11 +17,7 @@ const Headline = ({ blocks }) => {
     return null;
   }
 
-  return (
-    <StyledHeadline>
-      {text}
-    </StyledHeadline>
-  );
+  return <StyledHeadline>{text}</StyledHeadline>;
```

Comment:
The duplication highlighted here by CodeClimate will be addressed in a separate PR.